_sass/navbar.scss

```diff
@@ -1,36 +1,40 @@
 .nav-bar {
-    width: 100%;
-    background-color: #133667;
-    font-family: "Poppins", serif;
-    font-weight: 700 !important;
-    padding-top: 10px;
+  width: 100%;
+  background-color: #133667;
+  font-family: "Poppins", serif;
+  font-weight: 700 !important;
+  padding-top: 10px;
 }
 
 .nav-content .logo-touch-target {
-    display: inline-flex;
```

Comment:
This might conflict with @RishiDesai17 and @yahya-cloud navbar PR. Maybe check in with them once?